app/src/main/java/com/example/nicocalculator/MainActivity.kt

```diff
@@ -2,11 +2,29 @@ package com.example.nicocalculator
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
+    private val setInitValueOperations = "Operations"
```

Comment:
Please, declare kotlin constants as a "const val", either as a public constant in the beggining of the class, or inside of a private companion object. Also, try to follow naming conventions for constants and give a more discriptive name, for example: OPERATIONS_DEFAULT_TEXT

Reply:
Done

Reply:
Done